JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -99,19 +99,10 @@ fun ArticleScreen(
 
     Row(modifier.fillMaxSize()) {
         val context = LocalContext.current
-        if (showNavRail) {
-            ArticleNavRail(
```

Comment:
This is the main TODO: We want to replace this NavRail with a blank one, other than a back button.

Reply:
What do you mean with the blank one? This is needed for the article immersive screen but we'd want to have something custom and hide the top-level one. It's probably ok to remove this code now as it's not getting called anyway

Reply:
Discussed offline for the strategy for medium screens. If you don't mind, I'll do all of that in a follow up PR after this one.